doc/lua_api.txt

```diff
@@ -5380,6 +5380,10 @@ This is basically a reference to a C++ `ServerActiveObject`
     * in first person view
     * in third person view (max. values `{x=-10/10,y=-10,15,z=-5/5}`)
 * `get_eye_offset()`: returns `offset_first` and `offset_third`
+* `send_mapblock(blockpos)`:
+    * Sends a server-side loaded mapblock to the player.
+    * Returns `false` if failed.
+    * Resource intensive - use sparsely
```

Comment:
should say something like: to get the mapblock, simply divide the position by 16.
The engine hopefully would round automagically

Reply:
geeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeesh

Reply:
fixed with an exact and better description.

Reply:
I've actually added an explanation of getting nodepos from blockpos and blockpos from nodepos, elsewhere in lua_api.txt so no need to explain all that again.